external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceProvider.scala

```diff
@@ -107,46 +106,6 @@ private[kafka010] class KafkaSourceProvider extends DataSourceRegister
     new KafkaTable(strategy(options.asMap().asScala.toMap))
   }
 
-  /**
-   * Creates a [[org.apache.spark.sql.sources.v2.reader.streaming.ContinuousReadSupport]] to read
-   * Kafka data in a continuous streaming query.
-   */
-  override def createContinuousReadSupport(
```

Comment:
moved to https://github.com/apache/spark/pull/23619/files#diff-eeac5bdf3a1ecd7b9f8aaf10fff37f05R413